src/controllers/auth.controller.js

```diff
@@ -0,0 +1,182 @@
+const bcrypt = require('bcrypt');
+const { ApiError } = require('../exceptions/ApiError.js');
+const { userService } = require('../services/user.service.js');
+const { User } = require('../models/User.js');
+const { jwtService } = require('../services/jwt.service.js');
+const { tokenService } = require('../services/token.service.js');
+
+function validateEmail(email) {
+  if (!email) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(email)) {
+    return 'Email is not valid';
+  }
+}
+
+function validatePassword(password) {
+  if (!password) {
+    return 'Password is required';
+  }
+
+  if (password.length < 6) {
+    return 'At least 6 characters';
+  }
```

Comment:
The `validatePassword` function does not return a value when the password is valid. Consider returning `null` or `undefined` to indicate successful validation.